morphic.js

```diff
@@ -1270,7 +1270,7 @@ function copy(target) {
         }
     } else {
         c = {};
-        for (property in target) {
+        for (property in target) { /* eslint guard-for-in: 0 */
```

Comment:
How do we feel about comments like this for special cases in code? (Things like eval, etc). They should be rare, but if we want to be thorough, they will exist.